src/context.ts

```diff
@@ -2843,6 +2843,28 @@ export class Context implements RenamedUpdate {
         );
     }
 
+    /**
+     * Context-aware alias for `api.sendGift`. Sends a gift to the given channel chat. The gift can't be converted to Telegram Stars by the receiver. Returns True on success.
+     *
+     * @param gift_id Identifier of the gift
+     * @param other Optional remaining parameters, confer the official reference below
+     * @param signal Optional `AbortSignal` to cancel the request
+     *
+     * **Official reference:** https://core.telegram.org/bots/api#sendgift
+     */
+    replyWithGiftToChannel(
+        gift_id: string,
+        other?: Other<"sendGift", "user_id" | "chat_id" | "gift_id">,
+        signal?: AbortSignal,
+    ) {
+        return this.api.sendGift(
```

Comment:
@KnightNiwrem excellent catch
```suggestion
        return this.api.sendGiftToChannel(
```